### PR TITLE
Remove example pipelines

### PR DESCRIPTION
**Pull request by mrchtr**
This PR updates the README and removes the following example pipelines: 
- commoncrawl
- controlnet
- filter-cc-25m
- rag-cc-pipeline
- stable diffusion

Before we merge this, we should ensure that every use case repository has been published.

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
I made all the example repos public. Can you add a section to the README with the links?

**Comment by PhilippeMoussalli** [comment]
Could you also remove datacomp?

**Comment by mrchtr** [comment]
> Could you also remove datacomp?

I've removed the whole example folder and added the links to the new repositories.

File: README.md [hunk 1]
```diff
@@ -71,47 +71,12 @@ Anxious to get started? Here's is a [step by step guide](https://fondant.readthe
...
-
-We have published an [image dataset containing 25 million images](https://huggingface.co/datasets/fondant-ai/fondant-cc-25m).
-As a result, we have provided a [sample pipeline](https://github.com/ml6team/fondant/tree/main/examples/pipelines/filter-cc-25m) that
-demonstrates the download and filtering of these images. In the pipeline folder,
-you will find detailed instructions on how to execute the pipeline and explore the images.
-
-### Fine-tuning ControlNet
-
-Our
-[example pipeline to generate data for ControlNet fine-tuning](https://github.com/ml6team/fondant/tree/main/examples/pipelines/controlnet-interior-design)
```
Comment (RobbeSneyders) [inline README.md:86]: These images can be deleted as well.

**Review by PhilippeMoussalli — APPROVED** [review]
Thanks Matthias!

**Pull request merged**